modulesAndFiles.c

```diff
@@ -0,0 +1,107 @@
+﻿#include <stdio.h>
+#include <malloc.h>
+#include <stdbool.h>
+#include <string.h>
+#include <ctype.h>
+
+#include "errorCodes.h"
+#include "modulesAndFiles.h"
+
+char* readFile(int* const errorCode, const char* const fileName)
+{
+    size_t length = 0;
+    size_t capacity = 1;
+    FILE* file;

```

Comment:
```suggestion
    FILE* file = NULL;
```
Иначе неинициализированная переменная, что запрещено стайлгайдом